coursefunctions.h

```diff
@@ -80,6 +83,33 @@ void GameOfLife(Buffer2D<PIXEL> & target)
         if(!isSetup)
         {
                 // Your Code goes here
+                int neighbors = 0;
```

Comment:
This code isn't necessary for the rest of the class. I would consider just removing it.